docs/manage-data.md

```diff
@@ -0,0 +1,30 @@
+# Data management
+
+eoAPI-k8s provides a basic data ingestion process that consist of manual operations on the components of the stack.
+
+# Load data
+
+## Preshipped bash script
+
+Execute `make ingest` to load data into the eoAPI service.
+
+## Manual steps
+
+In order to add raster data to eoAPI you can load STAC collections and items into the PostgreSQL database using pgSTAC and the tool `pypgstac`.
+
+### Prerequisites
+
+- Ensure your Kubernetes cluster is running and `kubectl` is configured to access and modify it.
+- You have STAC records for the collection and items you wish to load (e.g., `collection.json` and `items.json`).
+
+### Import data
+
+On the pod or server running the raster eoAPI service, you can use the following commands to load the data:
+
+```bash
+#!/bin/bash
+apt update -y && apt install python3 python3-pip -y && pip install pypgstac[psycopg]';
+pypgstac pgready --dsn $PGADMIN_URI
+pypgstac load collections /tmp/collection.json --dsn $PGADMIN_URI --method insert_ignore
```

Comment:
Similarly, given this needs data mounted on `/tmp` we probably want to hand hold people into how to do that.

Reply:
Yes, indeed, thanks for raising this. I adjusted the docs.